_src/_posts/2018-12-19-jplotlib-help-your-teams-to-leverage-data-by-using-the-perfect-graphs.markdown

```diff
@@ -0,0 +1,89 @@
+So everyone can use an already customized, Jampp branded and easy to go Python library to introduce data results with no effort.
+
+<!--excerpt.end-->
+
+Before, with an open source library defaults...
+![ open source library default histogram ]({{site.url}}/assets/images/jplotlib/old_histogram.png){: .center-image }
+
+After, with JPlotLib:
+![ jplotlib histogram ]({{site.url}}/assets/images/jplotlib/histogram.png){: .center-image }
+
+You might be thinking: Why? Some libraries already have what you need.
+But that is not true.
+We create and analyze tailored data-sets, user behavior and patterns from millions and millions of data and let’s be honest, almost none of the default suggestions from softwares are the correct ones to show it.
+Some libraries have tons of documentation and bugs, so changing small details requires a lot of manual input, a.k.a “human-time”.
+
+Examples with before and after results and the small coding changes needed in the README are key for an easy and solid experience.
+
+## Extreme-Makeover, Jampp edition
+
+This is the part where we show you how different your visual experience can be by improving your plotting tools.
+
+### Bar Plot
+
+#### Before
+![ basic bar plot ]({{site.url}}/assets/images/jplotlib/old_bar.png){: .center-image }
+
+#### After
+![ jplotlib bar plot ]({{site.url}}/assets/images/jplotlib/bar.png){: .center-image }
+
+### Barh Plot
+
+#### Before
+![ basic barh plot ]({{site.url}}/assets/images/jplotlib/old_barh.png){: .center-image }
+
+#### After
+![ jplotlib barh plot ]({{site.url}}/assets/images/jplotlib/barh.png){: .center-image }
+
+### Scatter Plot
+
+#### Before
+![ basic scatter plot ]({{site.url}}/assets/images/jplotlib/old_scatter.png){: .center-image }
+
+#### After
+![ jplotlib barh plot ]({{site.url}}/assets/images/jplotlib/scatter.png){: .center-image }
+
+
+This wouldn't be complete without some of the houses favorites.
+Here are some examples of the `slope` and `venn` charts developed on the `JPlotLib` package.
+
+### Slope Plot
+![ jplotlib slope plot ]({{site.url}}/assets/images/jplotlib/slope.png){: .center-image }
+
+### Venn Diagram
+![ jplotlib venn diagram ]({{site.url}}/assets/images/jplotlib/venn.png){: .center-image }
+
+
+
+## Bonus Points
+
+You can preset your brand’s color palette predefine color maps for heatmaps.
+This ensures a clean and coherent look and your Marketing team will love you.
+
+![ colors ]({{site.url}}/assets/images/jplotlib/colors.png){: .center-image }
+![ colormaps ]({{site.url}}/assets/images/jplotlib/colormaps.png){: .center-image }
+
+## Wrapping Up / Keep in Mind
+
+So remember, it’s not enough to “get” the data, you also need to show it in a way that allows others to quickly grasp its meaning.
+You have to make sure everyone in the company understand what it means with the least effort possible, and the best way to do that it to make easy-to-read (and pretty, don't forget pretty) figures that show it's value.
```

Comment:
understand -> understands

`the best way to do that it` ->  `the best way to do that is`